integrations/standalone/tests/pageobjects/Table.ts

```diff
@@ -115,8 +115,10 @@ export class Cell {
   async edit(value: string) {
     const expression = this.locator.getByRole('textbox');
     await expect(expression).toHaveAttribute('aria-expanded', 'false');
-    await expression.click();
-    await CodeEditorUtil.type(this.page, value);
+    const code = new CodeEditor(this.page, expression, expression, this.locator);
+    await code.focus();
+    await code.clearContent();
+    await this.page.keyboard.type(value);

```

Comment:
code.fill(value) does something like this already

Reply:
fill does make a blur at the end

Reply:
I will make an own impl of the code editor for ScriptCell.